thread-random/src/Counter.java

```diff
@@ -0,0 +1,27 @@
+import java.util.Random;
+
+public class Counter implements Runnable {
+    private static int count = 0;
+    private int id;
+
+    public Counter(int id)
+    {
+        this.id = id;
+    }
+
+    @Override
+    public void run()
+    {
+        try {
+            int ms = new Random().nextInt(6) + 5;
```

Comment:
@tatevikb does you thread work only one time ?

Reply:
Yes, each thread was working only once. I've made changes, now it works several times until the end of 1 minute.

Reply:
@hhov88